src/dpi.erl

```diff
@@ -66,11 +71,31 @@ load_unsafe() ->
         Path -> Path
     end,
     case erlang:load_nif(filename:join(PrivDir, "dpi_nif"), 0) of
-        ok -> ok;
+        ok -> register_process(RemotePid);
         {error, {reload, _}} -> ok;
         {error, Error} -> {error, Error}
     end.
 
+-spec(pids_get() -> [pid()]).
+pids_get() ->
+    exit({nif_library_not_loaded, dpi, pids_get}).
+
+-spec(pids_set([pid()]) -> ok).
+pids_set(Pids) when is_list(Pids)  ->
+    exit({nif_library_not_loaded, dpi, pids_set}).
+
+register_process(Pid) ->
+    pids_set(
+        lists:filter(
+            fun(P) ->
+                true == (catch rpc:call(
```

Comment:
can be replaced with `rpc_call(node(P), erlang, is_process_alive, [P])` so the catch can be removed as well